src/models/GitHub.js

```diff
@@ -55,18 +56,28 @@ class GitHub extends Fetcher {
       date.setDate(date.getDate() - 31)
     }
     const dateStr = date.toISOString()
-    return this.get(`notifications?since=${encodeURIComponent(dateStr)}`)
+    return this.get(`notifications?since=${encodeURIComponent(dateStr)}`,
+                    { fetchAll: true }).then(result => result.json)
   }
 
   // https://developer.github.com/v3/search/#search-issues
   getTasks(query = Config.searchQuery) {
-    const urlPath = `search/issues?q=${encodeURIComponent(query)}`
-    return this.get(urlPath).then(({ items }) => items.map(d => getTask(d)))
+    const params = `?per_page=${this.perPage}&q=${encodeURIComponent(query)}`
+    const url = `${Config.githubApiUrl}/search/issues${params}`
+    return this.getTasksFromUrl(url)
+  }
+
+  getTasksFromUrl(url) {
+    return this.get(url).then(result => {
+      const { json, nextUrl } = result
```

Comment:
Yes! See https://developer.github.com/v3/#pagination. Each response that has a next page to it will provide a link to that next page of results in the header.